app/overlays/tgl/ipc4_native_drivers_overlay.conf

```diff
@@ -16,6 +16,8 @@ CONFIG_DAI_DMIC_HW_IOCLK=38400000
 CONFIG_DAI_DMIC_HAS_OWNERSHIP=y
 CONFIG_DAI_DMIC_HAS_MULTIPLE_LINE_SYNC=y
 
+CONFIG_INTEL_ADSP_IPC=y
```

Comment:
this is really not just rename :)

Reply:
heh yes, hard to fit all to the header line of commit, I try to explain it in commit message, but can also try to mod the title